

// try {
//     const error = new Error('This is a test error');
//     error.status = 400;
//     throw error;
// } catch (error) {
//     console.log('Error caught:', error.message);
//     console.log('Error status:', error.status);
// }

console.log(new Date());
